pkg/cmd/root/help_topic.go

```diff
@@ -42,16 +42,17 @@ var HelpTopics = []helpTopic{
 		name:  "environment",
 		short: "Environment variables that can be used with gh",
 		long: heredoc.Docf(`
-			%[1]sGH_TOKEN%[1]s, %[1]sGITHUB_TOKEN%[1]s (in order of precedence): an authentication token for github.com
-			API requests. Setting this avoids being prompted to authenticate and takes precedence over
-			previously stored credentials.
+			%[1]sGH_TOKEN%[1]s, %[1]sGITHUB_TOKEN%[1]s (in order of precedence): an authentication token that will be
+			used when the host that is targeted by a command is either github.com or a subdomain of ghe.com.
```

Comment:
I didn't know if shortening and using active voice was more in keeping with GitHub brand 🤔 not the end of the world but a consideration:

```suggestion
			used when a command targets either github.com or a subdomain of ghe.com hosts.
```

Reply:
The only other thing is some inconsistency with whether literal `github.com` and `ghe.com` should be bracketed with backticks, double quotes, etc.  Previously these were double quoted 🤷

Reply:
Eh I'm ok with active language but the sentence doesn't really read very well:

```
%[1]sGH_TOKEN%[1]s, %[1]sGITHUB_TOKEN%[1]s (in order of precedence): an authentication token that will be used when a command targets either github.com or a subdomain of ghe.com hosts.
```

How about:


```
%[1]sGH_TOKEN%[1]s, %[1]sGITHUB_TOKEN%[1]s (in order of precedence): an authentication token that will be used when a command targets a host that is either github.com or a subdomain of ghe.com
```

or

```
%[1]sGH_TOKEN%[1]s, %[1]sGITHUB_TOKEN%[1]s (in order of precedence): an authentication token that will be used when a command targets either github.com or a subdomain of ghe.com
```

Reply:
I like the last option ☝️

Reply:
We decided to go with the last one, so this now says:

```
`GH_TOKEN`, `GITHUB_TOKEN` (in order of precedence): an authentication token that will be used when
a command targets either github.com or a subdomain of ghe.com. Setting this avoids being prompted to
authenticate and takes precedence over previously stored credentials.
```

Thanks!